controller.go

```diff
@@ -102,6 +105,13 @@ func ControllerCommand() *cobra.Command {
 	billingEntityEmailSubject := cmd.Flags().String("billingentity-email-subject", "An APPUiO Billing Entity has been updated", "Subject for billing entity modification update mails")
 	billingEntityCronInterval := cmd.Flags().String("billingentity-email-cron-interval", "@every 1h", "Cron interval for how frequently billing entity update e-mails are sent")
 
+	saleOrderStorage := cmd.Flags().String("sale-order-storage", "none", "Type of sale order storage to use. Valid values are `none` and `odoo16`")
+	saleOrderClientReference := cmd.Flags().String("sale-order-client-reference", "APPUiO Cloud", "Default client reference to add to newly created sales orders.")
+	cmd.Flags().StringVar(&oc.URL, "sale-order-odoo16-url", "http://localhost:8069", "URL of the Odoo instance to use for sale orders")
```

Comment:
Please don't use version numbers of Odoo, as this could change without having to update something in this code here.

Reply:
We have those numbers everywhere.

It makes more sense to see Odoo16 as Odoo 16+ than having parameters like `odoo-old-old`, `odoo-new-old`, `odoo-old-new`, `odoo-new-new`

Reply:
Numbers are easy to understand and you see immediately which one is the most recent one. If you have any better parameter numbering system please propose here.